pkg/arvo/sys/vane/behn.hoon

```diff
@@ -389,18 +389,53 @@
 ++  scry
   |=  [fur=(unit (set monk)) ren=@tas why=shop syd=desk lot=coin tyl=path]
   ^-  (unit (unit cage))
+  ::  only respond for the local identity, %$ desk, current timestamp
   ::
-  ?.  ?=(%& -.why)
+  ?.  ?&  =(&+our why)
+          =([%$ %da now] lot)
+          =(%$ syd)
+      ==
     ~
-  ?.  ?=(%timers syd)
-    [~ ~]
-  =/  tiz=(list [@da duct])
+  ::  /bx/debug/timers  (list [@da duct])  all timers and their ducts
+  ::  /bx/timers        (list @da)         all timer timestamps
+  ::  /bx/timers/next   (unit @da)         the very next timer to fire
+  ::  /bx/timers/[da]   (list @da)         all timers up to and including da
+  ::
+  ?.  ?=(%x ren)  ~
```

Comment:
This seems right and proper to me. (I'm tempted to ask you to implement `%by` as well, but that's probably beest saved for a more systematic scry-cleanup effort.)